wallet-new/server/Cardano/Wallet/API/V1/LegacyHandlers/Addresses.hs

```diff
@@ -126,3 +127,11 @@ getAddress addrText = do
             accMod <- V0.txMempoolToModifier ws mps =<< V0.findKey accId
             let caddr = V0.getWAddress ws accMod adiWAddressMeta
             single <$> migrate caddr
+
+-- | Creates new address in external wallet.
+newExternalAddress
+    :: (MonadThrow m, V0.MonadWalletLogic ctx m)
+    => NewAddress
+    -> m (WalletResponse WalletAddress)
+newExternalAddress _ =
+    single <$> (liftIO $ generate arbitrary)
```

Comment:
same, let's get this to `undefined` so we don't accidentally get it into develop.